### ISSUE TITLE
Clearer output

### ISSUE THREAD

**Issue by nakosung** [ibab/tensorflow-wavenet#100]
Comparing to https://github.com/basveeling/wavenet example output, their implementation sounds much more clear.

Has anybody managed to produce clear output?

**Comment by lemonzi** [comment]
I would try with a different speaker, the seed audio at the beginning of
the example is already pretty noisy.

**Comment by ibab** [comment]
@nakosung: Is there any VCTK output example for that implementation? I couldn't find any.

**Comment by lemonzi** [comment]
Since we are only training in one speaker so far, we could look for a
shorter, very clean voice recording and train on that.

**Comment by nakosung** [comment]
Fast generation doesn't affect the quality of output? I haven't tried with fast generation disabled. btw, I'm training against single female voice (Korean voice corpus http://semanticweb.kaist.ac.kr/home/index.php/Voices_Corpus)

**Comment by ibab** [comment]
@nakosung: The fast generation produces identical results to the slow one. We have a unit test that compares the two.

**Comment by nakosung** [comment]
@ibab Great! :)

**Comment by nakosung** [comment]
Wave output from another wavenet implementation sounds more clear. https://github.com/ritheshkumar95/WaveNet

**Comment by adroit91** [comment]
@nakosung That sound is deliberately overfit on 8 speech sections of 8 seconds each from Blizzard dataset (this dataset is hard to get hands on too!) to test the memory capacity of the network, that's all.

**Comment by nakosung** [comment]
This is a generated output from korean corpus. I'm retrying with wider receptive field.
http://filebin.ca/2wnosmtNvIst/67200.wav

**Comment by nakosung** [comment]
It seems sophisticated network sounds better. Closing this issue. :)
